ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionLakarintygAvOgonspecialistMessage.java

```diff
@@ -68,4 +70,13 @@ public static CertificateDataElement toCertificate(int index, CertificateTextPro
             )
             .build();
     }
+
+    private static Boolean setVisibility(Syn syn) {
+        final var nattblindhet = syn != null ? syn.getNattblindhet() : null;
```

Comment:
For readability you could change this code to something like this:

    private static Boolean getVisibility(Syn syn) {
        if (syn == null) {
            return false;
        }
        return isTrue(syn.getNattblindhet()) || isTrue(syn.getProgressivOgonsjukdom()) || isTrue(syn.getSynfaltsdefekter());
    }

    private static Boolean isTrue(Boolean value) {
        return value != null ? value : false;
    }